src/pages/Loans/LoansOverview/components/LoansBaseTable/LoansBaseTable.style.tsx

```diff
@@ -37,4 +37,8 @@ const StyledTag = styled.div<StyledChipProps>`
   background-color: ${(props) => theme.transaction.status.bg[props.$variant]};
 `;
 
-export { StyledCellLabel, StyledCellSubLabel, StyledCellTickerLabel, StyledTag, StyledTitle };
+const StyledEmptyTableTitle = styled(P)`
+  font-weight: ${theme.fontWeight.bold};
+`;
```

Comment:
This could be replaced with `Strong` component from component-library -- no need for custom styled component